README.md

```diff
@@ -35,7 +35,7 @@ to `window`
 
 ## Basic Usage ##
 
-Here's a very simple example to get started. We have two pages `index.html` and `child.html`.This is the minimum you need to start get them talking to each other.
+Here's a very simple example to get started. We have two pages `index.html` and `child.html`.This is the minimum you need to get them talking to each other.
```

Comment:
```suggestion
Here's a very simple example to get started. We have two pages `index.html` and `child.html`. This is the minimum you need to get them talking to each other.
```

Reply:
@BaileyBrightman 
Just a heads up I think we lost this change in the mix somewhere.